templates/addons/cilium/cilium.yaml

```diff
@@ -15,8 +15,17 @@ spec:
     wait: true
     timeout: 5m
   valuesTemplate: |
+    routingMode: native
+    kubeProxyReplacement: true
+    ipv4NativeRoutingCIDR: 10.0.0.0/8
+    tunnelProtocol: ""
+    enableIPv4Masquerade: true
     ipam:
       mode: kubernetes
+    ipv4:
+      enabled: true
+    ipv6:
```

Comment:
not in scope for this PR, but do we know what the lift would be to enable dual stack networking?